docs/static-analysis.md

```diff
@@ -144,7 +144,7 @@ and [set up project specific settings and dependencies](setting-up.md#step-3-set
 1. The `Node interpreter` and `Stylelint package` should have been auto-filled to your locally installed NodeJS and
    `$PROJECT_DIR$/node_modules/stylelint` respectively. Point them to the right locations if they are not.
 1. Point `Configuration file:` to the location of `teammates-eslint.yml`.
-1. Under 'Extra eslint options:', add `--ext .es6 --ignore-pattern '**/*.js'`.
+1. Under 'Extra eslint options:', add `--ext .js --ignore-pattern 'src/main/webapp/js/*.js' 'src/main/webapp/test/*.js'`.
```

Comment:
Should we remove --ext .js since that is the default extension used?

Reply:
Actually, I realize if we minify everything for tests as well, IntelliJ can detect that the minified files and auto-ignore the file. Do you think it is a problem if we change it to minify everything?

Furthermore, if we ask the user to install the `Node.js` plugin `node_modules` will be auto-ignored.

Then we can remove the Extra eslint options altogether. What do you think?

Reply:
> if we minify everything for tests as well

We already did. Can you try first on your end if it works?

Reply:
Yeah, it seems to work for me.